telebot/apihelper.py

```diff
@@ -1152,6 +1152,15 @@ def set_chat_title(token, chat_id, title):
     return _make_request(token, method_url, params=payload, method='post')
 
 
+def set_my_description(token, description=None, language_code=None):
+    method_url = r'setMyDescription'
+    payload = {}
+    if description:
```

Comment:
HERE (only here) it's incorrect. :)

_"Pass an empty string to remove the dedicated description for the given language."_